app/controllers/parameters_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ParametersController < ApplicationController
+  before_action :find_parameter, only: %i[update destroy]
+
+  def create
+    @parameter = Parameter.new(parameter_params)
+
+    @parameter.save!
+
+    render json: @parameter
+  end
+
+  def update
+    @parameter.update(parameter_params)
```

Comment:
do we need to check for true/false?

Reply:
We essentially take whatever they have given us so we don't really need to check :).